scripts/curate_basel_additional_data.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+import json
+import argparse
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='Convert dataset to BIDS format.')
+    parser.add_argument("-i", "--path-input",
+                        help="Path to folder containing the dataset to convert to BIDS",
+                        required=True)
+    parser.add_argument("-o", "--path-output",
+                        help="Path to the output BIDS folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def main(path_input, path_output):
+    if os.path.isdir(path_output):
+        shutil.rmtree(path_output)
+    os.makedirs(path_output, exist_ok=True)
+    input_files = os.listdir(path_input)
+    input_files.sort()
```

Comment:
This is a bit easier

```suggestion
    input_files = sorted(os.listdir(path_input))
```